docs/api/en/renderers/WebGLRenderTarget.html

```diff
@@ -49,7 +49,8 @@ <h3>
 			<br />
 			[page:Boolean depthBuffer] - default is `true`. <br />
 			[page:Boolean stencilBuffer] - default is `false`.<br />
-			[page:Number samples] - default is `0`.<br /><br />
+			[page:Number samples] - default is `0`.<br />
+			[page:Number count] - The number of render targets. Default is `1`.<br /><br />
```

Comment:
This was copied and pasted from the MultipleRenderTargets docs, but looks out of context here. Is there any way to specify that this configures the number of color attachments without getting too technical?